modules/openshift-storage-common-terms.adoc

```diff
@@ -50,6 +50,22 @@ KMS key:: The Key Management Service (KMS) helps you achieve the required level
 
 Local volumes:: A local volume represents a mounted local storage device such as a disk, partition or directory.
 
+Nested mount points:: A nested mount point is a mount point that attempts to use a mount point created by a previous volume.
++
+For example:
++
+[source,terminal]
+----
+- mountPath: /opt/ibm/wlp/output
+  name: output-folder-pvc
+...
+- mountPath: /opt/ibm/wlp/output/defaultServer/resources/security/keystore/pkcs12/server.p12
+  name: key-trust-store
+  subPath: keystore/pkcs12/server.p12 
+----
++
+It is recommended that you do _not_ use nested mount points, since {product-title} does not guarantee the order in which mount points are created. Such usage is prone to race conditions and undefined behavior.
```

Comment:
```suggestion
Do _not_ use nested mount points, because {product-title} does not guarantee the order in which mount points are created. Such usage is prone to race conditions and undefined behavior.
```
- Minor suggestion for simplicity.
- ISG advises to not use "since" as a synonym for "because": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#since